[2025-04-28T06:45:29.510+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:45:29.521+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:45:29.522+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-28T06:45:29.578+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-04-28T06:45:29.587+0000] {standard_task_runner.py:57} INFO - Started process 372 to run task
[2025-04-28T06:45:29.591+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp0evdlpcw']
[2025-04-28T06:45:29.594+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask insert_worldmeter
[2025-04-28T06:45:29.651+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 77b690a9fde8
[2025-04-28T06:45:29.817+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-04-28T06:47:28.403+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-04-28T06:47:32.384+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-28T06:47:32.842+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250428T064529, end_date=20250428T064732
[2025-04-28T06:47:33.306+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-28T06:47:33.533+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-03T07:22:10.900+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-03T07:22:10.901+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-03T07:22:10.934+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-03T07:22:10.934+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-03T07:22:10.974+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-03T07:22:10.984+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-03T07:22:10.984+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-03T07:22:10.998+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-05-03T07:22:11.005+0000] {standard_task_runner.py:57} INFO - Started process 256 to run task
[2025-05-03T07:22:11.007+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpulj5mek8']
[2025-05-03T07:22:11.010+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask insert_worldmeter
[2025-05-03T07:22:11.071+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-03T07:22:11.072+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-03T07:22:11.074+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host f2b491f5d506
[2025-05-03T07:22:11.157+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-03T07:24:40.356+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-05-03T07:24:43.989+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-03T07:24:44.544+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250503T072210, end_date=20250503T072444
[2025-05-03T07:24:45.020+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-03T07:24:45.266+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-05T13:47:41.372+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-05T13:47:41.373+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-05T13:47:41.419+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-05T13:47:41.420+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-05T13:47:41.613+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-05T13:47:41.622+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-05T13:47:41.623+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-05T13:47:41.643+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-05-05T13:47:41.653+0000] {standard_task_runner.py:57} INFO - Started process 212 to run task
[2025-05-05T13:47:41.656+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp_3dpknyo']
[2025-05-05T13:47:41.660+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask insert_worldmeter
[2025-05-05T13:47:41.721+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-05T13:47:41.721+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-05T13:47:41.723+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 1cb5a07b27d6
[2025-05-05T13:47:41.816+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-05T13:49:57.642+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-05-05T13:50:01.289+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-05T13:50:01.779+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250505T134741, end_date=20250505T135001
[2025-05-05T13:50:02.272+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-05T13:50:02.580+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-10T17:03:07.733+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-10T17:03:07.736+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-10T17:03:07.785+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-10T17:03:07.786+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-10T17:03:07.848+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-10T17:03:07.861+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-10T17:03:07.862+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-10T17:03:07.879+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-05-10T17:03:07.893+0000] {standard_task_runner.py:57} INFO - Started process 525 to run task
[2025-05-10T17:03:07.897+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpmtzvgbvf']
[2025-05-10T17:03:07.900+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask insert_worldmeter
[2025-05-10T17:03:07.971+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-10T17:03:07.971+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-10T17:03:07.973+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 45059e99c3e8
[2025-05-10T17:03:08.115+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-10T17:06:10.429+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-05-10T17:06:14.186+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-10T17:06:14.713+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250510T170307, end_date=20250510T170614
[2025-05-10T17:06:15.299+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-10T17:06:15.669+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-11T16:06:03.318+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-11T16:06:03.319+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-11T16:06:03.358+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-11T16:06:03.359+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-11T16:06:03.414+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-11T16:06:03.427+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-11T16:06:03.428+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-11T16:06:03.448+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-05-11T16:06:03.456+0000] {standard_task_runner.py:57} INFO - Started process 272 to run task
[2025-05-11T16:06:03.461+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpkgkp6s2i']
[2025-05-11T16:06:03.464+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask insert_worldmeter
[2025-05-11T16:06:03.519+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-11T16:06:03.519+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-11T16:06:03.521+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 9a7df5060863
[2025-05-11T16:06:03.604+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-11T16:09:57.874+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-05-11T16:10:02.425+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-11T16:10:02.995+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250511T160603, end_date=20250511T161002
[2025-05-11T16:10:03.727+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-11T16:10:04.225+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-12T09:21:27.449+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-12T09:21:27.450+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-12T09:21:27.485+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-12T09:21:27.486+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-12T09:21:27.658+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-12T09:21:27.666+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-12T09:21:27.667+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-12T09:21:27.680+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-05-12T09:21:27.686+0000] {standard_task_runner.py:57} INFO - Started process 239 to run task
[2025-05-12T09:21:27.689+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpweqwqgbg']
[2025-05-12T09:21:27.691+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask insert_worldmeter
[2025-05-12T09:21:27.737+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-12T09:21:27.738+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-12T09:21:27.739+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 16fa90749134
[2025-05-12T09:21:27.813+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-12T09:24:05.884+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-05-12T09:24:09.928+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-12T09:24:10.421+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250512T092127, end_date=20250512T092410
[2025-05-12T09:24:10.970+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-12T09:24:11.219+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-13T12:50:07.507+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-13T12:50:07.508+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-13T12:50:07.542+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-13T12:50:07.542+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-13T12:50:07.583+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-13T12:50:07.592+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-13T12:50:07.593+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-13T12:50:07.605+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-05-13T12:50:07.613+0000] {standard_task_runner.py:57} INFO - Started process 657 to run task
[2025-05-13T12:50:07.617+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpdu1br4a_']
[2025-05-13T12:50:07.619+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask insert_worldmeter
[2025-05-13T12:50:07.669+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509
[2025-05-13T12:50:07.669+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_worldmeter'
[2025-05-13T12:50:07.671+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 1d46261ea2e6
[2025-05-13T12:50:07.751+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-13T12:53:28.679+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-05-13T12:53:32.660+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-13T12:53:33.297+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250513T125007, end_date=20250513T125333
[2025-05-13T12:53:34.090+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-13T12:53:34.439+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
